gr-vocoder/grc/vocoder_codec2_decode_ps.block.yml

```diff
@@ -5,7 +5,7 @@ flags: [ python, cpp ]
 parameters:
 -   id: mode
     label: Bit rate
-    dtype: raw
```

Comment:
iirc, having this as `raw` allows the value to be set as a variable in GRC but doesn't affect the selectability of it as an enum

Reply:
It's not just a warning. It deletes `option_attributes` if not `enum` so the C++ code here won't work if `raw`.